src/App.jsx

```diff
@@ -7,7 +7,7 @@ import Sidebar from './components/sidebar/Sidebar';
 import MainDashboard from './components/main-dashboard/MainDashboard';
 import { useUser } from './components/UserProvider';
 
-const geocodingURI = 'http://api.openweathermap.org/geo/1.0/direct?q=';
+const geocodingURI = 'https://api.openweathermap.org/geo/1.0/direct?q=';
```

Comment:
Is using HTTPS required for the deployment to work?

Reply:
It appears to be, I received a "mixed content" error when trying to load the page.

Reply:
> It appears to be, I received a "mixed content" error when trying to load the page.

https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content?utm_source=mozilla&utm_medium=firefox-console-errors&utm_campaign=default

Reply:
Oh okay I see, that is weird why that happened since I don't see how the API call should affect deployment. There's probably details I'm missing though. Sounds good!